Scripts/generate_release_zip.sh

```diff
@@ -169,9 +169,7 @@ fi
 echo "Make Objective-C framework zip file ..."
 mkdir -p "$OUTPUT_OBJC_DIR"
 cp -R "$BUILD_DIR/$SCHEME_PREFIX ObjC"/* "$OUTPUT_OBJC_DIR"
-if [[ -z ${WORKSPACE} ]]; then
-    cp Scripts/Support/License/LICENSE_${EDITION}.txt "$OUTPUT_OBJC_DIR"/LICENSE.txt
-else # official Jenkins build's license
+if [[ -n $WORKSPACE ]]; then
```

Comment:
Workspace will be populated if building from Jenkins, if not that means its been run by user itself(in case of CE), or by anyone who has permission to EE repo. Release builds are done only via Jenkins, which requires License.